CHANGES.md

```diff
@@ -22,6 +22,7 @@
 - Fixed handling of subtree root transforms in `Implicit3DTileContent`. [#9971](https://github.com/CesiumGS/cesium/pull/9971)
 - Fixed issue in `ModelExperimental` where indices were not the correct data type after draco decode. [#9974](https://github.com/CesiumGS/cesium/pull/9974)
 - Fixed WMS 1.3.0 `GetMap` `bbox` parameter so that it follows the axis ordering as defined in the EPSG database. [#9797](https://github.com/CesiumGS/cesium/pull/9797)
+- Fixed a typo in the `KmlDataSource` documentation. [#9976](https://github.com/CesiumGS/cesium/pull/9976)
```

Comment:
Small things like typos don't need to be mentioned in `CHANGES.md`

Reply:
Sounds good :+1: